pfcpiface/utils.go

```diff
@@ -44,3 +38,11 @@ func getOutboundIP(dstIP string) net.IP {
 
 	return localAddr.IP
 }
+
+func mySEID(peerSEID uint64) uint64 {
+	return (peerSEID >> 2)
+}
+
+func peerSEID(mySEID uint64) uint64 {
+	return (mySEID << 2)
+}
```

Comment:
Implement localSEID allocator function, [store remoteSEID in the session](https://github.com/wmnsk/go-gtp/blob/master/gtpv2/session.go#L115). Also [sequence number](https://github.com/wmnsk/go-gtp/blob/master/gtpv2/conn.go#L48) for messages.

Reply:
This will be done in the next PR.